components/admin_console/billing/plan_details.tsx

```diff
@@ -136,9 +142,11 @@ const PlanDetails: React.FC = () => {
     }
 
     let planPricing;
+    let planDetailsDescription;
+    let userCountDisplay;
 
     const showSeatsAndSubscriptionDates = false;
-    let userCountDisplay;
+    const goneBackUnderThreshold = userCount <= userLimit && (userCount + userInactiveCount) > userLimit;
```

Comment:
We don't take inacive users into account for user limits. If we want to see if they have gone back down from the threshold, we can just check `userCount <= userLimit` and that the subscription type is paid tier `susbscription.is_paid_tier == `true``. This would be the case that they have upgraded and but won't get charged because they are below the limit